res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+					</description>
+				</scripttemplate>
+				<scripttemplate index="10" product="2" licence_type="2" guid="c5c297fc-ecbb-4595-a342-40fc8f1d3b27">
+					<title>
+						<de>Zurück nach Hûrlýë Island</de>
+						<en>Back to Hûrlýë Island</en>
+					</title>
+					<description>
+						<de>Doch LeCruck ist inzwischen wieder zurückgesegelt, um Evelyn zu heiraten. Raymond muss die Kirche auf Hûrlýë Island stürmen, um das zu verhindern!</de>
+						<en>But LeCruck has since sailed back to marry Evelyn. Raymond must gatecrash the church on Hûrlýë Island to prevent this!</en>
+					</description>
+				</scripttemplate>
+				<scripttemplate index="11" product="2" licence_type="2" guid="a38b5c1a-ee98-49f8-83d0-06a3d27a5eaf">
+					<title>
+						<de>Das Elixier</de>
+						<en>The Elixir</en>
+					</title>
+					<description>
+						<de>Evelyn kann sich schon ganz gut selbst helfen. Und Raymond verliert auch noch das Elixier bei seiner tollpatschigen Rettungsaktion. Wie kann er nun im großen Showdown LeCruck doch noch besiegen?</de>
+						<en>Evelyn can already help herself quite well. And Raymond also loses the elixir during his clumsy rescue attempt. How can he still defeat LeCruck in the big showdown?</en>
+					</description>
+				</scripttemplate>
+			</children>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="2" required="1" gender="1" role_guid="6e1f331a-7c9d-433b-bc9f-7606ad322a5e" />
+				<job index="2" function="2" required="1" gender="2" role_guid="3ee1d44a-f3fc-47a5-aabf-25803e6e9f99" />
+				<job index="3" function="2" required="1" gender="1" role_guid="410a932c-9ceb-4080-9dbc-f82ac3c9bcd3" />
+				<job index="4" function="2" required="1" gender="2" />
+				<job index="5" function="2" required="1" gender="1" />
+				<job index="6" function="2" required="1" gender="2" />
+				<job index="7" function="2" required="1" gender="1" />
+			</jobs>
+
+			<genres maingenre="1" subgenres="5,10" />
+			<episodes min="12" max="12" />
+			<blocks value="1" />
+			<price min="80000" max="130000" />
+			<potential min="59" max="95" />
+			<outcome min="71" max="95" />
+			<review min="79" max="95" />
+			<speed min="54" max="89" />
+			<studio_size value="3" />
+		</scripttemplate>
+
+		<scripttemplate product="2" licence_type="3" guid="61661455-9607-44fd-a090-c1caa8ee3205" creator="9551" comment="inspired from https://en.wikipedia.org/wiki/Dead_Winter_Dead">
+			<title>
+				<de>%dead% %winter% in %sarajevo%</de>
+				<en>%dead% %winter% in %sarajevo%</en>
+			</title>
+			<description>
+				<de>Kriegsdrama über den jungen, enthusiastischen und leicht beeindruckbaren Stevan auf der einen, und die warmherzige, aber resolute Kara auf der anderen Seite der Front in Jugoslawien, und einem Teufels-Cellisten dazwischen.</de>
```

Comment:
If you remove the roles from the cast, the main character names could also be used as variables.

Reply:
I'm adding a todo about using variables, but I don't understand what you mean. Why do I have to remove roles? I thought I'll have to reference them from there?

Reply:
In script templates, I would like to use as few roles as possible. For me a template really is a template for different programmes, not one for producing the same think over and over.
Roles are really good if you want a connection between the results (different Spiderman movies). With variables you can add more variety - and you can reference different names just as well.

Reply:
If you have roles in it ... then this is not a big problem.
I am not sure when I added them - maybe it was before "variables" were existing.

Roles for now are for recurring elements. Stuff which can be independent from a casted person. So any famous movie role (Indiana Jones, Han Solo, James Bond) is what justifies a specific "role".
There are roles which are bound to the movie itself. So you won't see James Bond in a "non-James Bond"-Movie.
Most roles are kinda connected (indirectly, via movies) to franchises I guess (think of the whole Superhero-movie-stuff - which has different titles but the same "universe").

The role allows to have a centric way of naming a person. This guarantees a consistent use of a specific "spoof name" (Pan Pholo, Yams Pond) if the role itself is a kind of trademark and needs spoofing.

Reply:
> In script templates, I would like to use as few roles as possible.

For technical reasons?

I was intending to add a lot of roles to the global data once that is properly supported. For the `programme` entries but as well for scripts, as there are a lot of "famous" characters also in novels, computer games, etc...

So far I was creating role entries in my user file if
(1) the role is used in more than one script, and/or
(2) if the character is "famous" enough in some circles, and/or
(3) because it's just a nice game-play experience when you actually know and see a role name to which you assign an actor (hence my raised issue https://github.com/TVTower/TVTower/issues/835)

If it's not desired for technical or other reasons to create role entries in the database solely for reason (3), then maybe this could be mitigated if I could specify a `first_name` and `last_name` in the `job` directly (and perhaps also reference it in the text)?

Reply:
No, no technical reason As I said, it would be nice to have as many generic templates as possible - you define script templates, not (one-time) scripts. One template for many "different" movies. Varying character names and places at least give the illusion that you do not produce remakes over and over.

You would not define the names in the job but rather as regular variables (with options). And you can reference the same name in title/description of all episodes consistently.